src/frontend/js/index.js

```diff
@@ -919,21 +919,40 @@ const menuView = state => {
                         state.layers.od_matrices.variable !== "" && state.selectedZones.length == 0 && [m('br'), m('p', "(Click a zone to see outgoing flows)")],
 
                         state.selectedZones.length !== 0 && [
-                            m('label', {for: 'deselect_zone'}, 'Showing absolute flows to ', arrayToHumanList(state.selectedZones.map(id => zoneToHuman(id,state))), ' (deselect? ',
-                                m('input', {
+                            m('div', { class: 'flowlistholder' },
+                                m('span', 'Showing absolute flows for:'),
+                                m('ul', state.selectedZones.map(id => m('li',
+                                    m('input', {
```

Comment:
Maybe these would look better? https://vrimar.github.io/construct-ui/#/components/checkbox

I think they automatically get stacked one per line, so we could lose the special list CSS and stuff?